R/assign_missingness.R

```diff
@@ -152,25 +152,26 @@ from the conditions and assigned their missingness. The created comparisons are:
     dplyr::group_by(.data$comparison) %>%
     dplyr::mutate(n = dplyr::n()) %>%
     dplyr::filter(.data$n > 1) %>%
-    dplyr::mutate(n_replicates = paste0(.data$n_replicates, collapse = "/")) 
-  
-  if(any(unequal_replicates$n > 2)){
+    dplyr::mutate(n_replicates = paste0(.data$n_replicates, collapse = "/"))
+
+  if (any(unequal_replicates$n > 2)) {
```

Comment:
We could consider running the package `styler`as github action - so our code is always tidy: https://usethis.r-lib.org/reference/tidyverse.html - i will add it to the GitHub Discussions